src/data/finset/lattice.lean

```diff
@@ -1044,12 +1044,49 @@ end
   {f : α → β} (hf : monotone f) (s : finset α) (h : (s.image f).nonempty) :
   (s.image f).min' h = f (s.min' ((nonempty.image_iff f).mp h)) :=
 begin
-  refine le_antisymm (min'_le _ _ (mem_image.mpr ⟨_, min'_mem _ _, rfl⟩))
-    (le_min' _ _ _ (λ y hy, _)),
-  obtain ⟨x, hx, rfl⟩ := mem_image.mp hy,
-  exact hf (min'_le _ _ hx)
+  convert @max'_image αᵒᵈ βᵒᵈ _ _ (λ a : αᵒᵈ, to_dual (f (of_dual a))) (by simpa) _ _; convert h,
+  apply eq_iff_iff.mpr (nonempty.image_iff _).symm,
+end
+
+lemma max_eq_max' {s : finset α} (hs : s.nonempty) : s.max = s.max' hs :=
```

Comment:
Is the proof of this just [`finset.coe_sup'`](https://leanprover-community.github.io/mathlib_docs/data/finset/lattice.html#finset.coe_sup')?